event-buffer/event-buffer-core/src/main/java/uk/gov/justice/services/event/buffer/core/service/BufferInitialisationStrategyProducer.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.services.event.buffer.core.service;
+
+
+import uk.gov.justice.services.common.configuration.Value;
+import uk.gov.justice.services.event.buffer.core.repository.streamstatus.StreamStatusJdbcRepository;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class BufferInitialisationStrategyProducer {
+
+    private static final String TRUE = "true";
+    @Inject
+    @Value(key = "postgres.db", defaultValue = "true")
+    String usePostgresStrategy;
```

Comment:
I suggest we set the property to be a fully qualified class name. Then any strategy can be added via the classpath. If no value is provided, it can default to the ANSI strategy. Also suggest giving the property name more of a namespace, so it is clear where the property is used.